pkg/types/interfacetests/chain_reader_interface_tests.go

```diff
@@ -46,6 +47,7 @@ const AnyExtraValue = 3
 
 func RunChainReaderInterfaceTests[T TestingT[T]](t T, tester ChainReaderInterfaceTester[T]) {
 	t.Run("GetLatestValue for "+tester.Name(), func(t T) { runChainReaderGetLatestValueInterfaceTests(t, tester) })
+	t.Run("BatchGetLatestValue for "+tester.Name(), func(t T) { runChainReaderBatchGetLatestValueInterfaceTests(t, tester) })
```

Comment:
/nit `"BatchGetLatestValue for " + tester.Name(),`

Reply:
this is how Goland lints it and the linter doesn't complain 🤷

Reply:
Strings might get special treatment for readability since they often have leading or tailing whitespace :shrug:

Reply:
String concatenation gets condensed for me too. Seems like a common pattern.